docs/moderne-cli/README.md

```diff
@@ -43,13 +47,9 @@ Boot 2 to Spring Boot 3.
    </TabItem>
    </Tabs>
 
-  :::tip
-  On MacOS you can also use homebrew: `brew install moderneinc/moderne/mod` and skip step 4.
-  :::
-
-3. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
+4. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
```

Comment:
Why 4 ?
```suggestion
3. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
```

Reply:
Already reverted, for some reason github does this automatically when I save through "edit a file"